### DISCUSSION TITLE
Change login layout

### DISCUSSION THREAD

**Discussion by mateusgalasso** [laravel/nova-issues#3823] [Q&A] [answered]
Good morning, I need to change the layout of the Laravel Nova 4 login screen. How could I do that?

**Comment by michaelhume** [comment, marked as answer]
You can add the following to config/nova.php to override the login route.

```php
'routes' => [
        'login' => '/login'
    ],
```

**Reply by ramonfeleus** [reply]
Same here. We display a branded login page for a customer with their logo and a background image. Previously this could be done quite easy by overriding a blade file. How can that be done now?

**Reply by ramonfeleus** [reply]
It's not about overriding a route. But instead just the look and feel of the login page. Login handling still needs to remain the same, so how is that possible now? It seems it's not possible to extend a blade file, change html. And the login page does not have html id's or specific classes which could be targeted to add a background image or other details to give it some branding of our customer.

**Reply by juhotaipale** [reply]
I've managed this by installing Laravel Breeze which allows me to customize the login experience as much I want. Just override the login route and set redirect to Nova after successful login. Works like a charm!
